_posts/blog/2021-04-09_Trusted_Firmware_M_v_1.3.0_release.md

```diff
@@ -0,0 +1,39 @@
+---
+author: shebu.kuriakose
+title: Trusted Firmware-M v1.3 Released 
+date: 2020-07-22 10:00:00
```

Comment:
@dbharbin the most recent blogs won't show this post since the date is set to July 2020.